modules/zip/src/lib/tar/types.ts

```diff
@@ -0,0 +1,29 @@
+export type Structure = {
```

Comment:
These names are very generic and have no comments. How about

- Prefix these names with `Tar`...
- add a `/** ... */` comment before each type. Even the most simple explanation is better than nothing.